sdk/python/kfp/compiler/_gcs_helper.py

```diff
@@ -42,3 +42,13 @@ def remove_gcs_blob(gcs_path):
   def download_gcs_blob(local_path, gcs_path):
     blob = GCSHelper.get_blob_from_gcs_uri(gcs_path)
     blob.download_to_filename(local_path)
+
+  @staticmethod
+  def create_gcs_bucket_if_not_exist(gcs_bucket):
```

Comment:
add a docstring if it's a public method.

Reply:
Will do.